tests/sys/src/db/SqliteSetup.hx

```diff
@@ -0,0 +1,33 @@
+package db;
+
+import sys.db.Sqlite;
+import sys.db.Connection;
+import sys.FileSystem;
+
+class SqliteSetup extends utest.Test {
+	static public final DB_FILE = 'temp/db.sqlite';
```

Comment:
Suggestion: use an in-memory database (`DB_FILE = ':memory:'`) and avoid poluting the filesystem and having to manually delete the file and table for every test.

Reply:
Unfortunately `:memory:` drops the data once connection is closed. That makes it impossible to test #9700

Reply:
Changed to in-memory except for that test.

Reply:
Oh, right, I missed that.